### PR TITLE
nick/highfi-modal

### PR DESCRIPTION
**Pull request by niwong**
[//]: # "These comments meant for your reference, they are invisible and don't need to be deleted"

## What's new in this PR?

[//]: # '#####################################'
[//]: # '####### YOUR ANSWER BELOW ###########'
[//]: # '############## END ##################'

Implemented MOST OF the high-fidelity modal. Just need styling from @iris-hou on what the "Wants Dividends?" and "Number of Shares" form fields should look like.

### How to Review

[//]: # 'The order in which to review files and 
what to expect when testing locally'
[//]: # '#####################################'
[//]: # '####### YOUR ANSWER BELOW ###########'
[//]: # '############## END ##################'

Take a look at the React JSX/HTML in `AdminDashboard.js` then checkout the styles at the bottom of `AdminDashboard.js` I tried to use inline styling as little as possible.
Screen Shot 2020-03-13 at 10 25 45 PM

### Related PRs

[//]: # "Optional - related PRs you're waiting on
/ PRs that will conflict, etc"
[//]: # '#####################################'
[//]: # '####### YOUR ANSWER BELOW ###########'
[//]: # '############## END ##################'

### Next steps

[//]: # "What doesn't work yet, what's NOT in this 
PR/has to be done "
[//]: # '#####################################'
[//]: # '####### YOUR ANSWER BELOW ###########'
[//]: # '############## END ##################'

### Tests Performed, Edge Cases

[//]: # '#####################################'
[//]: # '####### YOUR ANSWER BELOW ###########'
[//]: # '############## END ##################'

### Screenshots

[//]: # '#####################################'
[//]: # '#### YOUR SCREENSHOTS BELOW ########'
[//]: # '############## END ##################'

CC: @dfangshuo

### CODE REVIEW

File: src/screens/admin/AdminDashboard.js [hunk 1]
```diff
@@ -293,26 +299,25 @@ class AdminDashboard extends React.Component {
...
                   <p>Project Group</p>
-                  <p>{projectGroup.name}</p>
+                  <p className="admin-project-group">{projectGroup.name}</p>
                 </div>
               </div>
               <div
                 className="admin-invite-form-row"
-                style={{ flexDirection: 'column', alignItems: 'center' }}
+                style={{
+                  flexDirection: 'column',
```
Comment (ashleynguyen27) [inline src/screens/admin/AdminDashboard.js]: maybe it would be nicer to extract inline styling

File: src/screens/admin/AdminDashboard.js [hunk 2]
```diff
@@ -293,26 +299,25 @@ class AdminDashboard extends React.Component {
...
-                      border: '1px solid var(--pp-black)',
-                      float: 'left',
-                      padding: '5px 10px',
-                      width: '100px'
-                    }}
+                    className="admin-invite-form-submit"
                   />
                 </div>
-                <p>{status}</p>
+                <h4 style={{ textAlign: 'center', margin: '0px' }}>{status}</h4>
```
Comment (ashleynguyen27) [inline src/screens/admin/AdminDashboard.js]: ^
